storybookjs/react-native
# feat: add ondevice actions for 6.0

Issue: closes #200 

## What I did

Added actions back to the example and implemented a temporary solution for the addon action preset that adds the actions from argtypes.

## How to test

1. Run the example
2. Go to the action example
3. press the button and open the addon panel and navigate to the actions tab to see the action logged there
